.circleci/config.yml

```diff
@@ -389,24 +391,24 @@ workflows:
   nightly:
     triggers:
       - schedule:
-          cron: "20 17 * * 0,3"
+          cron: "20 17 * * *"
           filters:
             branches:
               only:
                 - master
-                - /^feature-.*$/
     jobs:
-      - build-macos
-      - valgrind
+      - performance_ci_automation:
+          context: common
 
-  nightly_performance:
+  nightly-twice-a-week:
     triggers:
       - schedule:
-          cron: "20 17 * * *"
+          cron: "20 17 * * 0,3"
           filters:
             branches:
               only:
                 - master
+                - /^feature-.*$/
     jobs:
-      - performance_ci_automation:
-          context: common
+      - build-macos
```

Comment:
@rafie what about `performance_ci_automation`?

Reply:
@danni-m `performance_ci_automation` just runs under `nightly`. So there's `nightly` and `nightly-twice-a-week`.